app/controllers/veterans/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Veterans::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  # You should configure your model like this:
```

Comment:
Are the code comments included here useful? Are they a partial thrown away implementation?